build/chip/fuzz_test.gni

```
@@ -117,6 +117,14 @@ template("chip_pw_fuzz_target") {
       #   "MAKE_BUILD_TYPE=RelWithDebug",
       #  ]
 
+      if (use_coverage) {
+        if (defined(configs)) {
+          configs += [ "${build_root}/config/compiler:coverage" ]
+        } else {
+          configs = [ "${build_root}/config/compiler:coverage" ]
+        }
+      }
+
```

I wonder if this is shorter and repeats less:

```suggestion
      if (!defined(configs)) {
        configs = []
      }
      if (use_coverage) {
        configs += [ "${build_root}/config/compiler:coverage" ]
      }

```